spyplugins/ui/profiler/profiler.py

```diff
@@ -69,8 +72,9 @@ def get_plugin_title(self):
 
     def get_plugin_icon(self):
         """Return widget icon"""
-        return ima.icon('profiler')
-    
+        path = Path(self.PLUGIN_PATH) / self.IMG_PATH
```

Comment:
Please fix this: `self.PLUGIN_PATH` does not exist anymore :-) Its absence generates the following traceback when you try to open Preferences:

```python-traceback
Traceback (most recent call last):
  File "/home/carlos/Projects/spyder/github-repo/spyderlib/spyder.py", line 2663, in edit_preferences
    dlg.add_page(widget)
  File "/home/carlos/Projects/spyder/github-repo/spyderlib/plugins/configdialog.py", line 228, in add_page
    item.setIcon(widget.get_icon())
  File "/home/carlos/Projects/spyder/github-repo/spyderlib/plugins/__init__.py", line 58, in get_icon
    return self.plugin.get_plugin_icon()
  File "/home/carlos/Projects/spyder/github-repo/spyplugins/ui/pylint/pylint.py", line 99, in get_plugin_icon
    path = Path(self.PLUGIN_PATH) / self.IMG_PATH
AttributeError: 'Pylint' object has no attribute 'PLUGIN_PATH'
Traceback (most recent call last):
  File "/home/carlos/Projects/spyder/github-repo/spyderlib/spyder.py", line 2663, in edit_preferences
    dlg.add_page(widget)
  File "/home/carlos/Projects/spyder/github-repo/spyderlib/plugins/configdialog.py", line 228, in add_page
    item.setIcon(widget.get_icon())
  File "/home/carlos/Projects/spyder/github-repo/spyderlib/plugins/__init__.py", line 58, in get_icon
    return self.plugin.get_plugin_icon()
  File "/home/carlos/Projects/spyder/github-repo/spyplugins/ui/profiler/profiler.py", line 75, in get_plugin_icon
    path = Path(self.PLUGIN_PATH) / self.IMG_PATH
AttributeError: 'Profiler' object has no attribute 'PLUGIN_PATH'
```